bundles/tests/org.openhab.core.storage.mapdb.tests/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,6 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.8
+org.eclipse.jdt.core.compiler.compliance=1.8
+org.eclipse.jdt.core.compiler.problem.forbiddenReference=warning
+org.eclipse.jdt.core.compiler.release=disabled
```

Comment:
Is there a reason for adding these Eclipse project specific settings again? 
We removed them everywhere in https://github.com/eclipse/smarthome/pull/6106.

Reply:
No deliberate reason, it has been created by the Eclipse IDE automatically while importing the Maven project.

Reply:
done